File: Problemas/1772A - A+B/1772A_A+B.cpp
// 1772A - A+B
// https://codeforces.com/problemset/problem/1772/A

#include <stdio.h>
#include <stdlib.h>
#include <bits/stdc++.h>
#include <cmath>
using namespace std;

int main(){

    // Input:
    // 1) pruebas = Número de casos de prueba
    int pruebas;
    cin >> pruebas;

    // Itera sobre cada caso de prueba
    for(int indice_prueba = 0; indice_prueba < pruebas; indice_prueba++) {

        // Input:
        // 2) suma
        string suma;
        cin >> suma;

        // Convierte los dígitos A y B de string a entero
        // Resta su valor ASCII - 48 para obtener el valor entero
        int a = int(suma[0]) - 48;
        int b = int(suma[2]) - 48;

        // Realiza la suma
        int resultado = a + b;

        // Imprime el resultado de la suma
        cout << resultado << endl;
    }

    return 0;
}

File: Problemas/1772A - A+B/1772A_A+B.py
# 1772A - A+B
# https://codeforces.com/problemset/problem/1772/A

# Input:
# 1) pruebas = Número de casos de prueba
pruebas = int(input())

# Itera sobre cada caso de prueba
for indice_prueba in range(0,pruebas):

    # Input:
    # 2) suma = Suma a evaluar
    suma = input()

    # Realiza la suma
    # Castea los dígitos a enteros para sumarlos
    resultado = int(suma[0]) + int(suma[2])

    # Imprime el resultado de la suma
    print(resultado)